### PR TITLE
SONAR-6483 WS global_permissions/add_group add global permission to a group

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      .setHandler(this);
+
+    action.createParam(PARAM_PERMISSION)
+      .setDescription("Permission")
+      .setRequired(true)
+      .setPossibleValues(GlobalPermissions.ALL);
+
+    action.createParam(PARAM_GROUP_NAME)
+      .setRequired(true)
+      .setDescription("Group name or 'Anyone'")
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java]: Can a group be created with name "Anyone" ? If yes, then the conflict should be handled.

Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java]: no group can be created with name anyone, whatever the case. It's reserved.

Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java]: I suggest to support all case variants ("anyone", "Anyone", "ANYONE", ...)

Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java]: it's already the case, I'll update the description

File: server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import org.sonar.server.user.UserSession;
+
+public class AddGroupAction implements GlobalPermissionsWsAction {
+
+  public static final String ACTION = "add_group";
+  public static final String PARAM_PERMISSION = "permission";
+  public static final String PARAM_GROUP_NAME = "groupName";
+
+  private final InternalPermissionService permissionService;
+  private final UserSession userSession;
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/global/AddGroupAction.java]: MAJOR Remove this unused "userSession" private field. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request closed without merging**